RetailSuiteAccounting/Integration/07_ManageBWAReports.md

```diff
@@ -0,0 +1,139 @@
+
+The business evaluation (BWA) is a reporting system based on company financial data and provides information about the company's current cost and revenue situation and, consequently, its  profitability. BWA reports serve as a basis for managerial future decisions, and they are widely used by capital investors and credit institutions for company assessment. BWA is an essential planning and controlling tool for the company's future development.
+
+The *BWA assignment* tab in the *Accounting* module enables to create new BWA reports, to copy one of the predefined system reports and edit it, or to delete any of them. All available reports can be completely customized.
+
+ > [Warning] Be aware that any deletion is permanent and cannot be undone, and any changes made will overwrite the preset values.
+
+When customizing existing reports, bear in mind that the reports use the standard accounts set up in the *Accounting* module configuration process. For detailed information, see [Charts of accounts](./01_RunAccountingWizard.md#chart-of-accounts). If the accounts have been customized manually, they will have to be set up again individually for the BWA reports. Otherwise, the BWA reports will not recognized them.
+
+
+## Create a BWA report
+
+You can create a BWA report that is not yet available in the system.
+
+#### Prerequisites
+
+- The accounting is configured via the configuration wizard, see [Run the accounting wizard](./01_RunAccountingWizard.md).
+- A valid fiscal year is created, see [Create a fiscal year](./04_ManageFiscalYear.md#create-a-fiscal-year).
+
+#### Procedure
+
+*Accounting > Select the fiscal year > Settings > Tab BWA ASSIGNMENT*
+
+![Create a BWA report](../../Assets/Screenshots/RetailSuiteAccounting/Settings/BWAAssignment/CreateBWAReport.png "[Create a BWA report]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the right bottom corner.  
+The *Create* view is displayed.
+
+2. Enter a name in the *Report name* field.
+
+3. Click the [SAVE] button.  
+The new BWA report has been saved.
+
+  ![BWA report created](../../Assets/Screenshots/RetailSuiteAccounting/Settings/BWAAssignment/BWAReportCreated.png "[BWA report created]")
+
+4. Click the newly created report in the list.  
+The *Accounting report lines* section is displayed.
+
+  ![Add report lines](../../Assets/Screenshots/RetailSuiteAccounting/Settings/BWAAssignment/AddReportLines.png "[Add report lines]")
+
+5. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.  
+The *Create* section is displayed.
+
+  ![Create report lines](../../Assets/Screenshots/RetailSuiteAccounting/Settings/BWAAssignment/CreateReportLines02.png "[Create report lines]")
+
+6. Enter any relevant information in the corresponding fields. You can also add different formulas to it. For detailed information, see [BWA assignment](../UserInterface/02i_BWAAssignment.md).
```

Comment:
Das ist ein bisschen zu knapp formuliert. Ich weiß nicht, welche Felder ich ausfüllen muss und was da rein soll - auch nicht nach Lesen des UI Kapitels. Hier müssen noch mehr Schritte rein, die das genauer beschreiben. Eventuell Create a BWA report aufteilen in die Procedures Define the BWA report basic data und Add report lines